examples/resources/credential/cluster_proxy.tf

```diff
@@ -1,30 +1,59 @@
-# Create cluster proxy credential
-resource "tanzu-mission-control_credential" "proxy_cred" {
-  name = "proxy_cred"
+# Create explicit cluster proxy credential
+resource "tanzu-mission-control_credential" "explicit_proxy_cred" {
+  name = "explicit_proxy_cred"
 
   meta {
-    description = "proxy credential"
+    description = "explicit proxy credential"
     labels = {
       "key1" : "value1",
     }
     annotations = {
-      "httpProxy"  :"http://sfsdf.com:123",
-      "httpsProxy" :"http://sfsdf.com:123",
-      "noProxyList":"http://noproxy.com,http://something.com"
+      "proxyType" : "explicit",
+      "httpProxy" : "http://sfsdf.com:123",
+      "httpsProxy" : "http://sfsdf.com:123",
+      "noProxyList" : "http://noproxy.com,http://something.com"
     }
   }
 
   spec {
     capability = "PROXY_CONFIG"
-    provider = "GENERIC_KEY_VALUE"
+    provider   = "GENERIC_KEY_VALUE"
     data {
-      key_value{
+      key_value {
         data = {
           "httpUserName"  = "username"
           "httpPassword"  = "password"
           "httpsUserName" = "username"
           "httpsPassword" = "password"
-          "proxyCABundle" = "cabundle"
+          "proxyCABundle" = "-----BEGIN CERTIFICATE-----\n Encoded string for encryption of data\n ----END CERTIFICATE----" # chain of certificate is supported in CRT format
```

Comment:
Have you validated this case just to ensure there are no issues with reading the certificates?

Reply:
Yes I have, the certificate is read correctly in this format